app/controllers/leidinggevendenbeheer/bestuursfunctie/contact-info.js

```diff
@@ -0,0 +1,31 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class LeidinggevendenbeheerBestuursfunctieContactInfoController extends Controller {
+  @service router;
+
+  @tracked showConfirmationDialog = false;
+
+  get isDirty() {
```

Comment:
question: do we still use this? i'm guessing all editing is to be done in forms now?